pkg/test/resourcefixture/testdata/basic/monitoring/v1beta1/monitoringdashboard/monitoringdashboardrefs/create.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: monitoring.cnrm.cloud.google.com/v1beta1
+kind: MonitoringDashboard
+metadata:
+  name: monitoringdashboard-${uniqueId}
+spec:
+  projectRef:
+    external: ${projectId}
+  displayName: "monitoringdashboard-sample"
+  columnLayout:
+    columns:
+    - weight: 2
+      widgets:
+      - title: "Widget 1"
+        xyChart:
+          dataSets:
+          - timeSeriesQuery:
+              timeSeriesFilter:
+                filter: metric.type="agent.googleapis.com/nginx/connections/accepted_count"
+                aggregation:
+                  perSeriesAligner: "ALIGN_RATE"
+              unitOverride: "1"
+            plotType: LINE
+          timeshiftDuration: 0s
+          yAxis:
+            label: y1Axis
+            scale: LINEAR
+      - text:
+          content: "Widget 2"
+          format: "MARKDOWN"
+      - title: "Widget 3"
+        xyChart:
+          dataSets:
+          - timeSeriesQuery:
+              timeSeriesFilter:
+                filter: metric.type="agent.googleapis.com/nginx/connections/accepted_count"
+                aggregation:
+                  perSeriesAligner: ALIGN_RATE
+              unitOverride: "1"
+            plotType: "STACKED_BAR"
+          timeshiftDuration: 0s
+          yAxis:
+            label: y1Axis
+            scale: LINEAR
+      - title: "Widget 4"
+        logsPanel:
+          filter: metric.type="agent.googleapis.com/nginx/connections/accepted_count"
+          resourceNames:
+            - external: "projects/other${uniqueId}"
+              # TODO: Switch to a ref
```

Comment:
This is a bummer, although it opens a door.

Looks like logsPanel.resourceNames with a reference to a Project is broken in TF/DCL.

The good news is that because it's broken, I think we have more wiggle room in terms of whether kind should be required etc.

Sadly, the test is not very interesting because of this bug, but we'll fix that as part of going direct!